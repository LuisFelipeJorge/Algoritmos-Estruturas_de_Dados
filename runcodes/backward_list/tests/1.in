i 10 0
i 3 1
i 7 0
i 15 2
i 2 4
r 3
i 9 1
f
