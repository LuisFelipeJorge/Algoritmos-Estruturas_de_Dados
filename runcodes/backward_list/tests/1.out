[10,0] [7,2] [15,3] [2,4,0] [9,6,3]
